app/views/training_status/show.json.jbuilder

```diff
@@ -9,5 +9,13 @@ json.course do
     json.kind training_module.kind
     json.status training_progress_manager.status
     json.completion_date training_progress_manager.completion_date
+    tmu = TrainingModulesUsers.find_by(user_id: @user.id, training_module_id: training_module.id)
+    if training_progress_manager.completion_date
+      json.completion_time Time.at(tmu.completed_at - tmu.created_at)
+                               .utc
+                               .strftime('%H ' + I18n.t('users.training_module_time_field.hours')\
+                               + ' %M ' + I18n.t('users.training_module_time_field.minutes')\
+                               + ' %S ' + I18n.t('users.training_module_time_field.seconds'))
```

Comment:
This should probably be handled on the frontend. If `completion_time` is passed as a number (in seconds), then we can use `moment` to translate that into a human-friendly representation.

Doing it as three parts like this is going to lead to i18n problems with some languages, as the relationship between the hours, minutes and seconds strings is hard-coded (so right-to-left languages, and possibly others, will probably not get the right formatting).

Reply:
Done! Please have a look over it.